homeassistant/components/smarthab/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We need a strings.json which all the translations are going to be built from.

See `august/strings.json`